src/txn/replication_response_handler.cpp

```diff
@@ -71,6 +71,14 @@ void replication_response_handler(
     // but didn't have any values stored -- we use the default rep factor
     init_tier_replication(key_replication_map, tuple_key, key_tier);
     log->info("replication_response init_tier_replication tier {} key {}", key_tier, tuple_key);
+
+    if (error == AnnaError::KEY_DNE) {
+      AnnaError notify_error = AnnaError::NO_ERROR;
+      base_serializer->notify_dne_get(response.txn_id(), key, notify_error);
+      if (notify_error != AnnaError::NO_ERROR) {
+        // TODO: for some reason, the key exists now?
+      }
+    }
```

Comment:
This also seems to run on the first `START_TXN`. This might be a problem, since it creates a "GET" on some key even though no GET was requested.

Should I be filtering these responses more? E.g. tier != TXN